website/docs/r/tekton_pipeline.html.markdown

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
The names of the files can we have same convention as resource/datasource
Eg rename tekton_pipeline.html,markdown to cd_tekton_pipeline.html,markdown

Reply:
will address